src/basic/selinux-util.c

```diff
@@ -103,37 +107,94 @@ int mac_selinux_init(void) {
                   format_timespan(timespan, sizeof(timespan), after_timestamp - before_timestamp, 0),
                   (l+1023)/1024);
 
+        /* release memory after measurement */
+        if (label_hnd)
+                selabel_close(label_hnd);
+        label_hnd = TAKE_PTR(hnd);
+
+        return 0;
+}
+#endif
+
+int mac_selinux_init(void) {
+#if HAVE_SELINUX
+        int r;
+
+        if (initialized)
+                return 0;
+
+        if (!mac_selinux_use())
+                return 0;
+
+        r = selinux_status_open(/* no netlink fallback */ 0);
```

Comment:
Any particular reason why the netlink fallback is not used? This seems to cause failures in Fedora rawhide because the policy wasn't updated yet.

Reply:
Mostly because the status page exists since Linux 2.6.37 (and systemd requires 3.13).